VideoLocker/default_config/config.yaml

```diff
@@ -5,4 +5,4 @@ PLATFORM_DESTINATION_NAME: 'example.com'
 FEEDBACK_EMAIL_ADDRESS: 'support@example.com'
 COURSE_ENROLLMENT:
     ENABLED: false
-    EXTERNAL_COURSE_SEARCH_URL: 'http://10.0.2.2:8000/courses'
+    EXTERNAL_COURSE_SEARCH_URL: 'http://10.0.2.2:8000/courses'
```

Comment:
Might as well reinstate the newline at the end of the file here, to stop the diff viewers from complaining.